dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityServiceTest.java

```diff
@@ -570,26 +573,6 @@ void shouldReturnEmptyCollectionIfGivenFilterDoesNotMatch()
     assertIsEmpty(trackedEntities);
   }
 
-  @Test
-  void shouldReturnTrackedEntitiesGivenFilterOfOnlyUidEvenIfTrackedEntityDoesNotHaveGivenAttribute()
-      throws ForbiddenException, NotFoundException, BadRequestException {
-    TrackedEntityOperationParams operationParams =
-        TrackedEntityOperationParams.builder()
-            .organisationUnits(Set.of(orgUnitA.getUid(), orgUnitB.getUid()))
-            .orgUnitMode(DESCENDANTS)
-            .trackedEntityTypeUid(trackedEntityTypeA.getUid())
-            .filters(teaA.getUid())
-            .user(user)
-            .build();
-
-    List<TrackedEntity> trackedEntities =
-        trackedEntityService.getTrackedEntities(operationParams).getTrackedEntities();
-
-    // A filter with only a UID does not cause any join on attributes, so it does not actually
```

Comment:
Is that test removed because a filter with only a UID does actually cause an inner join? So this was a bug?

Reply:
It was just a duplicated test of `shouldReturnTrackedEntitiesIfTheyHaveGivenAttributeFilteredUsingOnlyUID` test